src/vm/costs/mod.rs

```diff
@@ -189,26 +218,38 @@ impl LimitedCostTracker {
             free: true,
         }
     }
-    pub fn load_boot_costs(&mut self) {
-        let boot_costs_id = boot_code_id("boot_costs");
+    pub fn load_boot_costs(&mut self, clarity_db: &mut ClarityDatabase) -> Result<()> {
+        let boot_costs_id = boot_code_id("costs");
```

Comment:
should `STACKS_BOOT_COST_CONTRACT` be used here instead?

Reply:
Yep, updated in 4242ee1